src/main/java/app/plugins/currenttime/AUtils.java

```diff
@@ -1,4 +1,4 @@
-package plugins.currenttime;
+package app.plugins.currenttime;
 
```

Comment:
plugins must contain class with the same _**name**_ and _**package**_:
`app.plugins.currenttime.AUtils` **!=** `app.plugins.greeting.AUtils`